ovos_tts_plugin_server/__init__.py

```diff
@@ -28,55 +38,76 @@ def v2(self) -> bool:
 
     @property
     def verify_ssl(self) -> bool:
+        """Whether or not to verify SSL certificates when connecting to the server. Defaults to True."""
         return self.config.get("verify_ssl", True)
 
-    def get_tts(self, sentence, wav_file, lang=None, voice=None):
-        lang = lang or self.lang
-        voice = voice or self.voice
-        params = {"lang": lang, "voice": voice}
+    def get_tts(
```

Comment:
I can read this better when at least `self` and another variable is in the same line as the `def`.  Not sure what it "should" be.

Reply:
This line is way too long due to the number of parameter and type hints. Also, it was auto-formatted by black. This is the "correct" way according to the formatter